examples/ThemeMaker.py

```diff
@@ -0,0 +1,271 @@
+            else:
+                button_light.configure(fg_color=None)
+            if (json_data["color"][i])[1]!=null:    
+                button_dark.configure(fg_color=(json_data["color"][i])[1])
+            else:
+                button_dark.configure(fg_color=None)
+                
+#Porgram widgets
+frame_info = customtkinter.CTkFrame(master=self, width=350, height=60)
+frame_info.place(x=20,y=20)
+
+widget_type = customtkinter.CTkLabel(master=frame_info,text=current, corner_radius=10, width=200, height=23,
+                                     fg_color=("white", "gray38"))
+widget_type.place(x=75,y=20)
+
+left_button= customtkinter.CTkButton(master=frame_info, text="<--", width=20, height=20, corner_radius=10,
+                                     fg_color=("white", "gray38"), command=ChangeModeLeft)
+left_button.place(x=20,y=20)
+
+right_button = customtkinter.CTkButton(master=frame_info, text="-->", width=20, height=20, corner_radius=10,
+                                       fg_color=("white", "gray38"), command=ChangeModeRight)
+right_button.place(x=290,y=20)
+
+menu=customtkinter.CTkOptionMenu(master=self,width=350,fg_color=("white", "gray38"), button_color=("white", "gray38"),
+                                 button_hover_color=None,
+                                 height=30, values=list0, command=update)
+menu.place(x=20,y=100)
+
+#Color chooser
+def changecolor():
+    color1=askcolor(title="Choose color for "+menu.get()+" (Light)")[1]
+    if color1!=None:
+        button_light.configure(fg_color=color1)
+        for i in json_data["color"]:
+            if i==menu.get():
+                (json_data["color"][i])[0]=color1
+def changecolor2():
+    color2=askcolor(title="Choose color for "+menu.get()+" (Dark)")[1]
+    if color2!=None:
+        button_dark.configure(fg_color=color2)
+        for i in json_data["color"]:
+            if i==menu.get():
+                (json_data["color"][i])[1]=color2
+                
+button_light = customtkinter.CTkButton(master=self, height=100, width=170, corner_radius=10, border_color="white",
+                                       fg_color=None, border_width=2, text="Light", hover_color=None, command=changecolor)
+button_light.place(x=20,y=150)
+
+button_dark = customtkinter.CTkButton(master=self, height=100, width=170, corner_radius=10, border_color="white",
+                                       fg_color=None, border_width=2, text="Dark", hover_color=None, command=changecolor2)
+button_dark.place(x=200,y=150)
+
+#Export the json file
+def savethis():
+    dialog=customtkinter.CTkInputDialog(master=self, text="Enter your theme name:", title="Save Theme")
+    try:
+        themename=dialog.get_input()+".json"
+        outfile=open(themename,"w")
+        json.dump(json_data, outfile, indent=2)
+        outfile.close()
+        tkinter.messagebox.showinfo("Exported!","Theme exported successfully!")
+    except:
+        pass

```

Comment:
Maybe add a tkinter error message

Reply:
It also shows error when we click cancel/close the input dialog, what to do?

Reply:
You need to check if `dialog.get_input()` retunes a string before appending the `.json`.

Reply:
@TomSchimansky Done, I have improve the error handling in both save and load functions, 
I think its ready to merge. Any more changes?

Reply:
I have removed the CtkInputDialog with tkinter savefile dialog. Now its working properly and makes more sense.